[Bug]: amfora build failure

### Problem description

https://github.com/johndebia2022/termux-packages/actions/runs/15287060200/job/42999434703

### What steps will reproduce the bug?

build

### What is the expected behavior?

_No response_

### System information

```shell
NA
```

<details><summary>Click here to see the build error output</summary>

```
go: github.com/makeworld-the-better-one/amfora/config imports
	code.rocketnine.space/tslocum/cview: unrecognized import path "code.rocketnine.space/tslocum/cview": parse https://code.rocketnine.space/tslocum/cview?go-get=1: no go-import meta tags (meta tag codeberg.org/tslocum/cview did not match import path code.rocketnine.space/tslocum/cview)
go: github.com/makeworld-the-better-one/amfora/client imports
	github.com/makeworld-the-better-one/go-gemini tested by
	github.com/makeworld-the-better-one/go-gemini.test imports
	github.com/google/go-cmp/cmp: code.rocketnine.space/tslocum/cview@v1.5.6-0.20210530175404-7e8817f20bdc: unrecognized import path "code.rocketnine.space/tslocum/cview": parse https://code.rocketnine.space/tslocum/cview?go-get=1: no go-import meta tags (meta tag codeberg.org/tslocum/cview did not match import path code.rocketnine.space/tslocum/cview)
go: github.com/makeworld-the-better-one/amfora/client imports
	github.com/spf13/viper imports
	github.com/spf13/cast tested by
	github.com/spf13/cast.test imports
	github.com/frankban/quicktest: code.rocketnine.space/tslocum/cview@v1.5.6-0.20210530175404-7e8817f20bdc: unrecognized import path "code.rocketnine.space/tslocum/cview": parse https://code.rocketnine.space/tslocum/cview?go-get=1: no go-import meta tags (meta tag codeberg.org/tslocum/cview did not match import path code.rocketnine.space/tslocum/cview)
go: github.com/makeworld-the-better-one/amfora/cache tested by
	github.com/makeworld-the-better-one/amfora/cache.test imports
	github.com/stretchr/testify/assert imports
	gopkg.in/yaml.v3 tested by
	gopkg.in/yaml.v3.test imports
	gopkg.in/check.v1 imports
	github.com/kr/pretty: code.rocketnine.space/tslocum/cview@v1.5.6-0.20210530175404-7e8817f20bdc: unrecognized import path "code.rocketnine.space/tslocum/cview": parse https://code.rocketnine.space/tslocum/cview?go-get=1: no go-import meta tags (meta tag codeberg.org/tslocum/cview did not match import path code.rocketnine.space/tslocum/cview)
Error: Process completed with exit code 1.
```

</details>

--- a/packages/amfora/0001-Add-replace-directive-for-cview-360.patch
+++ b/packages/amfora/0001-Add-replace-directive-for-cview-360.patch
@@ -1,0 +1,38 @@
+From cf5d091f442b150105f1395ce8d76529fa9f118f Mon Sep 17 00:00:00 2001
+From: Henry Firth <121139610+h14h@users.noreply.github.com>
+Date: Wed, 28 May 2025 08:49:18 -0500
+Subject: [PATCH] Add replace directive for cview (#360)
+
+---
+ go.mod | 2 ++
+ go.sum | 4 ++--
+ 2 files changed, 4 insertions(+), 2 deletions(-)
+
+diff --git a/go.mod b/go.mod
+index 882250c..1d30992 100644
+--- a/go.mod
++++ b/go.mod
+@@ -56,3 +56,5 @@ require (
+ 	gopkg.in/ini.v1 v1.67.0 // indirect
+ 	gopkg.in/yaml.v3 v3.0.1 // indirect
+ )
++
++replace code.rocketnine.space/tslocum/cview => codeberg.org/tslocum/cview v1.5.6-0.20210530175404-7e8817f20bdc
+diff --git a/go.sum b/go.sum
+index 7de0f07..031029d 100644
+--- a/go.sum
++++ b/go.sum
+@@ -37,8 +37,8 @@ cloud.google.com/go/storage v1.10.0/go.mod h1:FLPqc6j+Ki4BU591ie1oL6qBQGu2Bl/tZ9
+ cloud.google.com/go/storage v1.14.0/go.mod h1:GrKmX003DSIwi9o29oFT7YDnHYwZoctc3fOKtUw0Xmo=
+ code.rocketnine.space/tslocum/cbind v0.1.5 h1:i6NkeLLNPNMS4NWNi3302Ay3zSU6MrqOT+yJskiodxE=
+ code.rocketnine.space/tslocum/cbind v0.1.5/go.mod h1:LtfqJTzM7qhg88nAvNhx+VnTjZ0SXBJtxBObbfBWo/M=
+-code.rocketnine.space/tslocum/cview v1.5.6-0.20210530175404-7e8817f20bdc h1:nAcBp7ZCWHpa8fHpynCbULDTAZgPQv28+Z+QnhnFG7E=
+-code.rocketnine.space/tslocum/cview v1.5.6-0.20210530175404-7e8817f20bdc/go.mod h1:KBRxzIsj8bfgFpnMpkGVoxsrPUvnQsRnX29XJ2yzB6M=
++codeberg.org/tslocum/cview v1.5.6-0.20210530175404-7e8817f20bdc h1:e/aOXAZQlclmYpKW33oa6VZRFc/PA6O/70ijdAZJmo4=
++codeberg.org/tslocum/cview v1.5.6-0.20210530175404-7e8817f20bdc/go.mod h1:KBRxzIsj8bfgFpnMpkGVoxsrPUvnQsRnX29XJ2yzB6M=
+ dmitri.shuralyov.com/gpu/mtl v0.0.0-20190408044501-666a987793e9/go.mod h1:H6x//7gZCb22OMCxBHrMx7a5I7Hp++hsVxbQ4BYO7hU=
+ github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+ github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802/go.mod h1:IVnqGOEym/WlBOVXweHU+Q+/VP0lqqI8lqeDx9IjBqo=
+-- 
+2.49.0
+

--- a/packages/amfora/build.sh
+++ b/packages/amfora/build.sh
@@ -1,10 +1,10 @@
-TERMUX_PKG_HOMEPAGE=https://github.com/makeworld-the-better-one/amfora
+TERMUX_PKG_HOMEPAGE=https://github.com/makew0rld/amfora
 TERMUX_PKG_DESCRIPTION="Aims to be the best looking Gemini client"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="1.10.0"
-TERMUX_PKG_REVISION=1
-TERMUX_PKG_SRCURL=https://github.com/makeworld-the-better-one/amfora/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_REVISION=2
+TERMUX_PKG_SRCURL=https://github.com/makew0rld/amfora/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=0bc9964ccefb3ea0d66944231492f66c3b0009ab0040e19cc115d0b4cd9b8078
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true

